resources/serverless/charts/docker-registry/values.yaml

```diff
@@ -56,9 +56,14 @@ persistence:
   accessMode: 'ReadWriteOnce'
   enabled: true
   size: 20Gi
+  deleteEnabled: true
```

Comment:
Why do we use two variables to control the deletion?
Can we enable `garbageCollection` and disable `deleteEnabled`?
I think that we should use `garbageCollection` only to control this behavior.

Reply:
Fixed. Thank you for catching this. `deleteEnabled` was referenced in the deployment manifest, so I just added it and forgot to rename it later.